parsing/g_constr.mlg

```diff
@@ -438,13 +443,20 @@ GRAMMAR EXTEND Gram
         { List.map (fun (n, b, t) -> CLocalAssum ([n], Generalized (MaxImplicit, b), t)) tc }
       | "`["; tc = LIST1 typeclass_constraint SEP "," ; "]" ->
         { List.map (fun (n, b, t) -> CLocalAssum ([n], Generalized (NonMaxImplicit, b), t)) tc }
-      | "'"; p = pattern LEVEL "0" ->
-        { let (p, ty) =
-            match p.CAst.v with
-            | CPatCast (p, ty) -> (p, Some ty)
-            | _ -> (p, None)
-          in
-          [CLocalPattern (CAst.make ~loc (p, ty))] } ] ]
+      | "'"; p = pattern LEVEL "0" -> { [CLocalPattern p] } ] ]
+  ;
+  one_open_binder:
```

Comment:
I didn't see where one_open_binder or one_closed_binder are used.  Would you run `make doc_gram_rsts` and include those productions in an appropriate place in the manual?

Reply:
They are called in `vernac/egramcoq.ml`. They are supposed to be visible only with `Print Grammar constr` (or `Print Custom Grammar foo`) for the notations having the `binder` modifier. They are of somehow internal use and it is not clear whether it is important to document them.